modules/ROOT/pages/legacy-search-experience.adoc

```diff
@@ -0,0 +1,311 @@
+** Method type (GET, POST, PUT)
+** Description
+* Query parameters
+** Name
+** Description
+* URI parameters
+** Name
+* Types
+** Name
+** Properties
+* Traits
+* Properties
+* Security schemas
+
+These property names are based on RAML specifications. Equivalent properties are indexed for OAS specifications. For example, RAML uses base URI but OAS uses host and path.
+
+This example shows an API specification:
+
+[source,text,linenums]
+----
+#%RAML 1.0
+---
+title: Leagues API
+version: v1
+baseUri: https://leagues.com/api/
+
+types:
+ Team:
+   properties:
+     name: string
+     city: string
+
+/leagues/{leagueId}:
+ displayName: League resource
+ description: Contains all the information about a league
+ uriParameters:
+   leagueId:
+     displayName: League ID
+     description: The ID of the league to retrieve information from
+ /teams:
+   get:
+     displayName: Teams directory
+     description: Get list of all teams in the league
+     queryParameters:
+       offset:
+         displayName: Offset
+         description: Offset for the result set
+     responses:
+       200:
+         body:
+           application/json:
+               type: Team[]
+   post:
+     displayName: Create team
+     description: Creates a new team in the league
+     responses:
+       201:
+         description: New team has been created
+----
+
+This API is found by these searches:
+[%header%autowidth.spread]
+|===
+|Search term |Matches
+|league API |API title
+|leagues.com |API baseUri
+|`/leagues` |Endpoint path
+|League resource |Endpoint display name
+|Information about a league |Endpoint description
+|POST |Endpoint method
+|create team |Endpoint method description
+|offset |Query parameter name, Query parameter description
+|leagueid |URI parameter name
+|team |Type name
+|city |Type property name
+|leagues leagueId teams |Endpoint path
+|`/leagues/LEAGUE_ID/teams` |Endpoint path
+|===
+
+Note that the last two searches are the same, because special characters are removed before searching.
+
+== Search for Phrases 
+
+You can search for case-insensitive text that begins with each word you specify, put a space between words. This is similar to putting the `*` wildcard regular expression after each word. For example, searching for `mq module` finds any asset that has text that starts with `mq` or `module`.
+
+Searching for a multiple word phrase shows assets that have every word in the phrase.
+
+If you separate search words with any non-alphanumeric character other than a space, Anypoint Exchange converts the character to a space before searching.
+
+
+=== Search Separators
+
+The search experience uses search separators to find Exchange assets that contain one or more terms.
+
+If you input search words connected by non-alphanumeric symbols, Exchange replaces the symbols with spaces before searching for the term. The search string only succeeds if both words are present in the result. This is the same as the `word1 AND word2` relationship.
```

Comment:
Need a noun after "This"

Relationship seems strange here - maybe:

The result is equivalent to using `word1 AND word2`.

Reply:
agree and fixed